experiment/seccomp/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "seccomp"
```

Comment:
There is already a crate on crates.io named seccomp ; maybe we should rename this to something else if we are considering publishing this eventually, maybe seccomp-rs ?

Reply:
Yes, I know. I gonna change the name when it is published.